watertap/unit_models/tests/test_reverse_osmosis_0D.py

```diff
@@ -33,7 +33,7 @@
     PressureChangeType,
 )
 
-from watertap.unit_models.reverse_osmosis_base import TransportModel
+from watertap.unit_models.reverse_osmosis_base import TransportModel, ModuleType
```

Comment:
I think you should add some logic relating the ModuleType config to FrictionFactor config. You can start this by adding a test that intentionally sets the two to different values (when we know they should be the same) and see what you get. After that experiment, you could consider either replacing the friction factor config (partially or completely) or ensuring the user couldn't specify an incorrect combo of the two config options.

Reply:
@zacharybinger alternatively, since you're strictly trying to give an alternative for the effective area calculation, maybe instead of what I suggest above, you can just rename this new config option to something more specific like `ModuleAreaCalculation`. That way, it doesn't seem strange if I want to specify `spiral_wound` for my area but for whatever reason want to maintain the `flat_sheet` option for friction factor.

Does that make sense?

Reply:
@adam-a-a Personally I lean towards just having the one config option for `ModuleType` and having it apply to both the area calculation and the friction factor correlations. What do you think?

Reply:
I was originally leaning in that direction as well, but open to hear alternative ideas from others (e.g., @ElmiraShamlou @luohezhiming @TimBartholomew @avdudchenko )

Reply:
Building on my previous comments, I'm inclined to suggest that we should not categorize total membrane area and available membrane area for flux based on module type. A more effective approach might be to focus on the differences attributed to feed spacer width in our cross-sectional area calculations. This could potentially offer a more straightforward and universal method, applicable across various module types without the complexity of categorizing by each type.